lighthouse-core/audits/unused-css-rules.js

```diff
@@ -110,11 +111,16 @@ class UnusedCSSRules extends Audit {
       }
     }
 
-    return {
-      url: stylesheetInfo.header.sourceURL || 'inline',
-      label: `${percentUsed}% rules used`,
-      code: contentPreview.trim(),
-    };
+    let code;
+    let url = stylesheetInfo.header.sourceURL;
+    const label = `${percentUsed}% rules used`;
+
+    if (!url || url === pageUrl) {
```

Comment:
One thing I was curious about for later. When is url blank vs. the same as the page? Can you add a  comment?